src/index.js

```diff
@@ -0,0 +1,101 @@
+const skyLeft = {
```

Comment:
Great job setting up these data structures. This approach lets us handle responding to the sky selection changes without need to write large `if`/`else` conditions!

Consider combining all three of these structures to ensure that all the data pieces are provided together. Rather than three dictionaries with strings as values, we could make one dictionary using objects as the value, such as 
```js
Sunny: { left: "☀️🌤🌞🌤☀️", right: "☀️🌤🌞🌤☀️", color: "skyblue"}
```